models/model.js

```diff
@@ -38,4 +38,20 @@ exports.fetchArticlebyIdFromDB = (id) => {
         }
         else return results.rows[0];
     })
+}
+
+exports.fetchCommentsByIdFromDB = (id) => {
+    const queryString = `
+    SELECT comment_id, votes, created_at, author, body, article_id
+    FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC;
+    `;
+    return db.query(queryString, [id])
+    .then(results => {
+      if (results.rows.length === 0) {
```

Comment:
This length check will definitely be required however you may need an additional querty or to invoke another model here to check whether the comments are empty becuase there is none or whether its because the article doesn't exist